.env.example

```
@@ -1,10 +1,29 @@
 # Frontend API Configuration
 NEXT_PUBLIC_API_URL=http://localhost:8000
 
+# Backend runtime configuration
+# Required before importing/running api.main:app
+# Local/dev SQLite file example for the current API database layer
+DATABASE_URL=sqlite:///./dev.db
+SECRET_KEY=replace-with-a-long-random-secret
+
+# Required when seeding a new local/dev API database
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=replace-with-a-strong-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD key should go before the ADMIN_USERNAME key

(UnorderedKey)